tasks/main.yml

```
@@ -7,6 +7,15 @@
     name:
       - exim4
       - heirloom-mailx
+  when: ansible_distribution != "Debian" or (ansible_distribution == "Debian" and ansible_distribution_major_version < "10")
```

I think this ends up being a lexicographical comparison ("10" < "9") or at least that's not entirely clear. Probably safer to cast to int:
```suggestion
  when: ansible_distribution != "Debian" or (ansible_distribution == "Debian" and ansible_distribution_major_version|int < 10)
```